src/js/summary.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  update(newSummary) {
+    document.getElementById(`password-length`).innerHTML = newSummary.password.length;
```

Comment:
Let's move all DOM selectors to top of module. To not recalculate them on every `update`